src/actions/txNoteActions.js

```diff
@@ -110,15 +116,23 @@ export const getTxNoteByContactAction = (username: string) => {
       }
     });
 
-    await webSocketMessagesReceived
```

Comment:
We're not putting tx-note messages to reducer anymore, they're being decoded on the fly once received.